.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,32 @@
+name: Feature Request
+description: File a feature request
+title: ":pray: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We appreciate you taking the time to fill out this feature request!
+  - type: textarea
+    id: request
+    attributes:
+      label: What did you have in mind?
```

Comment:
The label and description feel like they're asking separate but equally important questions. I'm curious if you think we should streamline? I personally find the description more specific

Reply:
We can definitely streamline. I was using the description as an opener question to help them along, but maybe it's too far off the mark. I think we could change the label from a question to something closer to the ID. Would that work?

Reply:
Yes, I think that strategy works. I like the concept of leading them along, a few just felt significantly different to me.